tools/dockcross-cmake-builder.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if (( $# >= 1 )); then
+    image=$1
+    build_file=build-${image%:*}
+    shift 1
+
+    cmake_arg=$*
+    echo "cmake arg: $cmake_arg"
+
+    #echo "Pulling dockcross/$image"
+    #docker pull dockcross/"$image"
+
+    echo "Make script dockcross-$image"
+    docker run --rm dockcross/"$image" > ./dockcross-"$image"
+    chmod +x ./dockcross-"$image"
```

Comment:
github actions fails here because it doesn't have the permission to the docker repo.
```
docker: Error response from daemon: pull access denied for dockcross/windows-armv7, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.```